### PR TITLE
Ntate/feature/docr ea support

### PR DESCRIPTION
**Pull request by nicktate**
#### What does this pull request accomplish

Updates doctl to support current routes that will be offered at the EA release.

#### Dependencies

* [x] https://github.com/digitalocean/godo/pull/325
* [x] https://github.com/digitalocean/godo/pull/326

### CODE REVIEW

File: commands/registry.go [hunk 1]
```diff
@@ -259,9 +329,123 @@ func RunRegistryLogout(c *CmdConfig) error {
...
+	repository := c.Args[0]
+	tags := c.Args[1:]
+
+	if !force && AskForConfirm(fmt.Sprintf("delete %d repository tag(s)", len(tags))) != nil {
+		return fmt.Errorf("operation aborted")
+	}
+
+	for _, tag := range tags {
+		if err := c.Registry().DeleteTag(registry.Name, repository, tag); err != nil {
+			return err
```
Comment (adamwg) [inline commands/registry.go:393]: Might be more user-friendly to aggregate errors and return them at the end, so that one deletion failing in the middle of a larger set of tags doesn't block deleting the rest. Not sure if there's any pattern/precedent to follow from other subcommands.

File: commands/registry.go [hunk 2]
```diff
@@ -259,9 +329,123 @@ func RunRegistryLogout(c *CmdConfig) error {
...
+
+	registry, err := c.Registry().Get()
+	if err != nil {
+		return fmt.Errorf("failed to get registry: %w", err)
+	}
+
+	repository := c.Args[0]
+	digests := c.Args[1:]
+
+	if !force && AskForConfirm(fmt.Sprintf("delete %d repository manifest(s) by digest", len(digests))) != nil {
```
Comment (adamwg) [inline commands/registry.go:419]: At risk of making the confirmation too wordy, it would be nice to include a warning here that any associated tags will also be deleted.

File: commands/registry.go (hunk 1 above)
Comment (nicktate) [reply commands/registry.go:393]: I had considered this when initially implementing but after a quick (randomized) survey of other commands and deletion functionality, they all exit on first encounter of a failure without processing the rest.

Thoughts @hilary ?

Comment (bentranter) [reply commands/registry.go:393]: How likely is it that a deletion will fail? If it's pretty rare, I think it's okay to keep this as-is.

Comment (nicktate) [reply commands/registry.go]: Deletion failure would be pretty rare. 

I agree the experience would be a little nicer not blocking on failures. Since it sounds like we are okay with that process, I opted to update.
